README.md

```diff
@@ -1,16 +1,82 @@
 # RebbleOS
 
-RebbleOS is an open-source reimplementation of the firmware for the devices
-formerly manufactured by Pebble Technologies, Inc.  The firmware is based on
-FreeRTOS, and aims to be binary-compatible with applications that were
-written for the original Pebble OS, as well as radio-compatible with
-smartphone applications that are designed to work with Pebble.
+RebbleOS is an open-source reimplementation of the firmware for the devices formerly manufactured by Pebble Technologies, Inc.
+The firmware is based on FreeRTOS, and aims to be binary-compatible with applications that were written for the original Pebble OS, as well as radio-compatible with smartphone applications that are designed to work with Pebble.
+
+#### Links
+
+- https://github.com/ginge/FreeRTOS-Pebble
+- https://github.com/pebble-dev/wiki/wiki
+- https://github.com/pebble-dev/RebbleOS/issues
+
+## Supported Feature Matrix
+
+|Feature          |TinTin (Pebble 1)|Snowy (Time, Time Steel)|Chalk (Round)|Silk (Pebble 2)|
+|:---:            |:---:            |:---:                   |:---:        |:---:          |
+|Display:         |N                |Y                       |N            |N              |
+|Flash:           |Y                |Y                       |Y            |N              |
+|Buttons:         |Y                |Y                       |Y            |N              |
+|Gyro:            |N                |N                       |N            |N              |
+|Magnetometer:    |N                |N                       |N            |N              |
+|Vibrate:         |N                |Y                       |Y            |N              |
+|Backlight:       |N                |Y                       |Y            |N              |
+|Power Management:|N                |N                       |N            |N              |
+|Bluetooth:       |N                |Y                       |N            |N              |
+|Overall (%):     |1%               |20%                     |10%          |0%             |
+
+## Roadmap
```

Comment:
I think this section is too aspirational at the moment. Not much of the below is correct any more (and the prefix [PR] in he gist was the state of that particular task. I'm not sure tracking that sort of stuff in the readme is ideal.
Remove whole section please